docs/apache-airflow/start/docker-compose.yaml

```diff
@@ -150,6 +150,19 @@ services:
       _AIRFLOW_WWW_USER_USERNAME: ${_AIRFLOW_WWW_USER_USERNAME:-airflow}
       _AIRFLOW_WWW_USER_PASSWORD: ${_AIRFLOW_WWW_USER_PASSWORD:-airflow}
 
+  airflow-cli:
+    <<: *airflow-common
+    profiles:
```

Comment:
This container does not run by default
https://docs.docker.com/compose/profiles/